support-frontend/assets/components/subscriptionCheckouts/payPalSubmitButton.jsx

```diff
@@ -84,17 +65,12 @@ function SubscriptionSubmitButtons(props: PropTypes) {
             billingPeriod={props.billingPeriod}
           />
         </div>
-        <Button
-          id="qa-submit-button"
```

Comment:
could be worth running the post-deploy tests locally with this change just to double check!